### PR TITLE
[event] add end of August devenv sprint

### PR DESCRIPTION
**Pull request by j8xixo12**
Update late August devenv sprint page
but I haven't figured out the title..

### CODE REVIEW

File: content/pages/sprint/2021/08end-devenv.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
+============================================
+Scisprint 2021 End of August: solvcon devenv
```
Comment (yungyuc) [inline content/pages/sprint/2021/08end-devenv.rst:2]: How about use "Late August"?

File: content/pages/sprint/2021/08end-devenv.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+============================================
+
+:date: 2021-07-25 14:15
+:url: sprint/2021/08end-devenv
+:save_as: sprint/2021/08end-devenv.html
+
+Scisprint 2021 End of August: solvcon devenv
```
Comment (yungyuc) [inline content/pages/sprint/2021/08end-devenv.rst:9]: Maybe this sprint can host multiple projects, including swportal and Alan's bot.

File: content/pages/sprint/2021/08end-devenv.rst [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+possible to do it all by a single team.  External dependencies are necessary.
+However, because of the requirements of accuracy and performance, the
+dependencies are usually at the source level.  Building dependencies from
+source is the norm.  The project devenv is made to help the source-level
+dependency management and alignment for the runtime among developers.
+
+Registration
+------------
+
+Please join `gather.town <https://gather.town/app/yLTe8mBDb8pogMOX/sciwork>`_
```
Comment (yungyuc) [inline content/pages/sprint/2021/08end-devenv.rst:35]: If we do multple projects, perhaps this sprint should use a kktix for registration.

**Comment by yungyuc** [comment]
This draft is a good starting point for discussions.  Thanks!  I As I mentioned in the code review, I'd like to include Terry and Alan for their projects.  We should discuss with them.

File: content/pages/sprint/2021/08end-devenv.rst (hunk 3 above)
Comment (j8xixo12) [reply content/pages/sprint/2021/08end-devenv.rst]: Ok. Let me study how to use kktix for registration.
Does Sciwork have official kktix account?

**Comment by Alan-Kuan** [comment]
# Subtopic
Make an interactive event (sprint) scheduling and notifying Discord bot

# Plans
| Session | Duration |
| :---: | :---: |
| Self-introduction | 2min |
| Idea Sharing | 5min |
| Concept Demo | 5min |
| Discussion | 15min |

File: content/pages/sprint/2021/08end-devenv.rst (hunk 3 above)
Comment (yungyuc) [reply content/pages/sprint/2021/08end-devenv.rst]: Yes we do: https://sciwork.kktix.cc .  What's your kktix account?

Comment (j8xixo12) [reply content/pages/sprint/2021/08end-devenv.rst]: kktix account: j8xixo12

Comment (yungyuc) [reply content/pages/sprint/2021/08end-devenv.rst]: @j8xixo12 Can you provide me the email you use for the kktix account.  kktix seems to only allow me to invite using email.  Sorry about that.

Comment (j8xixo12) [reply content/pages/sprint/2021/08end-devenv.rst]: okok, email: `as2266317@gmail.com`

Comment (yungyuc) [reply content/pages/sprint/2021/08end-devenv.rst]: Thanks.  Invitation sent.

**Comment by yungyuc** [comment]
Second amendment: https://github.com/j8xixo12/swportal/pull/2

**Comment by yungyuc** [comment]
OK let's merge it and start to discuss #77 .

**Pull request merged**